frontend/src/views/WheresMyGene/components/GeneSearchBar/index.tsx

```diff
@@ -140,6 +151,28 @@ export default function GeneSearchBar({
           analyticsEvent={EVENTS.WMG_SELECT_GENE}
         />
 
+        {/* Clear Genes button */}
+        {!selectedGenes.length || (
+          <StyledButtonWrapper
+            onClick={() => {
+              if (dispatch) {
+                track(EVENTS.WMG_CLEAR_GENES_CLICKED);
+
+                dispatch(deleteAllGenes());
+              }
+            }}
```

Comment:
Thanks for raising this! 

I was reading https://kentcdodds.com/blog/usememo-and-usecallback and it seems like it only makes sense to prevent re-rendering if we know the computational cost is high upfront and/or we see noticeable perf issue

So in this case, if the re-rendering is not causing perf issue it's okay to not `useCallback()`!